lemarche/api/urls.py

```diff
@@ -39,6 +40,8 @@
         name="old_api_siae_siret",
     ),
     path("inbound-email-parsing/", InboundParsingEmailView.as_view(), name="inbound-email-parsing"),
+
+    path("datacube-tenders/", SimpleTenderList.as_view(), name="datacube-tenders"),
```

Comment:
```suggestion
    path("datacube/tenders/", SimpleTenderList.as_view(), name="datacube-tenders"),
```
you expect other endpoints in the future, no ?

Reply:
Hmmm, pas vraiment prévu à court terme, ça dépendra des besoins BI du marché à vrai dire...

Reply:
je rejoins l'avis de Raphael, je pense qu'on sollicitera d'autres cas d'usages de BI à l'avenir :)